test/e2e/specs/specs-playwright/wp-signup__wpcc.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @group calypso-release
+ */
+
+import {
+	DataHelper,
+	LoginPage,
+	setupHooks,
+	UserSignupPage,
+	CloseAccountFlow,
+	ReaderPage,
+	SupportComponent,
+	GutenboardingFlow,
+	EmailClient,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe(
+	DataHelper.createSuiteTitle( 'Signup: WordPress.com WPCC/Reader/Support Popover' ),
+	function () {
+		const inboxId = DataHelper.config.get( 'signupInboxId' ) as string;
+		const username = `e2eflowtestingwpcc${ DataHelper.getTimestamp() }`;
+		const email = DataHelper.getTestEmailAddress( {
+			inboxId: inboxId,
+			prefix: username,
+		} );
+		const signupPassword = DataHelper.config.get( 'passwordForNewTestSignUps' ) as string;
+
+		let page: Page;
+
+		setupHooks( ( args ) => {
+			page = args.page;
+		} );
+
+		describe( 'Signup via /start/wpcc', function () {
+			let activationLink: string;
+
+			it( 'Navigate to CrowdSignal and click to try for free', async function () {
+				//  No POM for this page.
+				await page.goto( 'https://crowdsignal.com/' );
+				await Promise.all( [ page.waitForNavigation(), page.click( 'a:text("Try it free")' ) ] );
+			} );
+
+			it( 'Create a new WordPress.com account', async function () {
+				const userSignupPage = new UserSignupPage( page );
+				await userSignupPage.signupWPCC( email, signupPassword );
```

Comment:
I love the crowdsignal integration, but I also think that this means that we can't run this test against a calypso.live, or our `wpcalypso.wordpress.com` endpoint because this will always go to wordpress.com.

I think we might have two options here:
1. Can we just jump directly to a WPCC auth route on the `calypsoBaseUrl`, and just bypass the crowdsignal piece of this? As cool as crowdsignal integration is, it might not also be a bad idea to limit dependencies during the pre-release execution time
2. We could probably intercept the link, parse it, swap out the host, and then go to right there. That would keep the crowdsignal integration intact, but allow us to use the right host client!

Reply:
> As cool as crowdsignal integration is, it might not also be a bad idea to limit dependencies during the pre-release exec

Yeah, that's an excellent critique. Currently, this _likely_ ends up running against production, which would be...pointless.

Reply:
There's this link (as per FG) that could serve as the direct link to the WPCC auth route: https://wordpress.com/start/crowdsignal/oauth2-name?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D34604bf9bcb69d07819b97c65b744925c061ed944e4d30623b9b8cebf1a4ce34%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dsignup%2526action%253Drequest_access_token%26wpcom_connect%3D1%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login&oauth2_client_id=978

Reply:
oh wowsers! Generating all those query params on the fly might be a bit onerous to maintain... I think we could just start by looking at the link and swapping out the hostname. I think we could run it for a bit a see if the crowdsignal integration ends up causing flakiness?

Reply:
I verified that `wpcalypso.wordpress.com/start/crowdsignal/...` works and loads the `wpcalypso` version of the webapp. 

@bsessions85 @kwight for lack of better folks to ask - do you see an issue with the approach used to test WPCC signup in this PR? 

The script is hitting the WPCC endpoint for CrowdSignal, fill out information and submit. This creates an account for the user, and the script goes on to try a few more things before closing the account.